crates/ty_python_semantic/resources/mdtest/import/module_getattr.md

```diff
@@ -0,0 +1,79 @@
+# Module-level `__getattr__`
+
+## Basic functionality
+
+```py
+import module_with_getattr
+
+# Should work: module `__getattr__` returns 'hi'
+reveal_type(module_with_getattr.whatever)  # revealed: str
+```
+
+`module_with_getattr.py`:
+
+```py
+def __getattr__(name: str) -> str:
+    if name == "whatever":
+        return "hi"
+    raise AttributeError(f"module has no attribute '{name}'")
```

Comment:
minor nit: I think this internal implementation of `__getattr__` is unnecessarily complex, because none of it has any impact on the type system's understanding of this `__getattr__`, and it raises questions for the reader that aren't really part of the purpose of this test, such as, would the type checker emit an error if some attribute other than `"whatever"` were accessed on this module? (Answer, we would not and cannot, but this test somehow implies that we might.)
```suggestion
    return "hi"
```